.storybook/components/SelectStory.js

```diff
@@ -5,13 +5,27 @@ import SelectItem from '../../components/SelectItem';
 import SelectItemGroup from '../../components/SelectItemGroup';
 
 const selectProps = {
-  onBlur: () => { action('blur'); }, // eslint-disable-line no-console
-  onClick: () => { action('click'); }, // eslint-disable-line no-console
-  onFocus: () => { action('focus'); }, // eslint-disable-line no-console
-  onMouseDown: () => { action('mouseDown'); }, // eslint-disable-line no-console
-  onMouseEnter: () => { action('mouseEnter'); }, // eslint-disable-line no-console
-  onMouseLeave: () => { action('mouseLeave'); }, // eslint-disable-line no-console
-  onMouseUp: () => { action('mouseUp'); }, // eslint-disable-line no-console
+  onBlur: () => {
+    action('blur');
+  }, // eslint-disable-line no-console
+  onClick: () => {
+    action('click');
+  }, // eslint-disable-line no-console
+  onFocus: () => {
+    action('focus');
+  }, // eslint-disable-line no-console
+  onMouseDown: () => {
+    action('mouseDown');
+  }, // eslint-disable-line no-console
+  onMouseEnter: () => {
+    action('mouseEnter');
+  }, // eslint-disable-line no-console
+  onMouseLeave: () => {
+    action('mouseLeave');
+  }, // eslint-disable-line no-console
+  onMouseUp: () => {
+    action('mouseUp');
+  }, // eslint-disable-line no-console
```

Comment:
IMO, you can remove all of these events since `onChange` is the only event that's used.